docs/website/docs/destinations.md

```diff
@@ -155,9 +155,9 @@ python3 chess.py
 
 ### Destination Configuration
 
-By default, a DuckDB database will be created inside the pipeline working directory with a name `quack.duckdb`. It is available in `read/write` mode via `pipeline.sql_client`.
+By default, a DuckDB database will be created in the current working directory with a name `quack.duckdb`. After loading, it is available in `read/write` mode via `with pipeline.sql_client() as con:` which is a wrapper over `DuckDBPyConnection`. See [duckdb docs](https://duckdb.org/docs/api/python/overview#persistent-storage) for details.
```

Comment:
@rudolfix is the name `quack.duckdb` or `<pipeline>.duckdb`?

Reply:
it should be pipeline name. I will fix it and let you review the docs again

Reply:
@TyDunn done